main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative './app'
+
+def main
+  new_app = App.new
+  new_app.run
+end
+
```

Comment:
- [OPTIONAL] The `main.rb` file is the entry point that is invoked at the end of the file. You should Present the user with a list of options to perform here and not in the app.js file.
- This method should do and look similar to the following:

```ruby
def main
  app = App.new()

  response = nil

  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
       ...
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()
```